cloud/shared/bin/lib/config_loader.py

```diff
@@ -70,6 +71,27 @@ def _load_config_fields(self, config_file: str):
         """
         config_parser = ConfigParser()
         config_fields = config_parser.parse_config(config_file)
+
+        if config_fields.get(
+                "ALLOW_ADMIN_WRITEABLE") and not os.getenv('SKIP_USER_INPUT'):
+            msg = inspect.cleandoc(
+                """
+                ###########################################################################
+                                                WARNING                                                       
+                ###########################################################################
+                ALLOW_ADMIN_WRITEABLE is set to true in your deploy config. This should only 
+                be set to true if this is the first time you are deploying and you want the custom
+                values for ADMIN_WRITEABLE to be applied. If this is not the case, 
+                please remove ALLOW_ADMIN_WRITEABLE and any ADMIN_WRITEABLE variables from 
+                your deploy config and update ADMIN_WRITEABLE variables via the admin settings
+                panel.
+
+                Would you like to continue with the deploy? [y/N] > 
+                """)
+            answer = input(msg)
+            if answer not in ['y', 'Y', 'yes']:
```

Comment:
**suggestion:** You could consider updating to `answer.lower() not in ['y', 'yes']`, so we accept any variation of these